client/controllers/AssignmentController.js

```diff
@@ -19,6 +19,13 @@ export function AssignmentController(
         store.dispatch(actions.initStore())
         store.dispatch(actions.loadAssignments('All', null, 'Created', 'Asc'))
         .then(() => {
+            const desks = sdPlanningStore.getDesksService()
```

Comment:
Do we need `sdPlanningStore.getDesksService()` method? Can we inject `desks` service into the controller?